crates/turbopack/examples/turbopack.rs

```diff
@@ -11,7 +11,7 @@ use std::{
 
```

Comment:
this file is in the folder `examples` not sure what the intention here?

Reply:
was expecting smth similar to this


https://github.com/vercel/turbo/tree/main/examples